cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -551,6 +564,135 @@ public synchronized boolean containsKey(PartialPath key) {
     }
   }
 
+  private class RemotePathAliasCache extends LRUCache<String, List<PartialPath>> {
+    private long valueSizeThreshold;
+    private AtomicLong valueSize = new AtomicLong(0);
+
+    // period executor thread
+    private ScheduledExecutorService periodExecutorService;
+
+    // executor every partition group update cache
+    private ExecutorService executorService = Executors.newFixedThreadPool(4);
```

Comment:
Why the core size is 4? Any special considerations？

Reply:
Reference Values in Most Scenarios

Reply:
Ok